frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Form, FormTitle, Icon, TitleInput, MessageInput, SubmitInput } from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const [postState, setPostState] = useState({});
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setPostState((prevState) => ({ ...prevState, [name]: value }));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(postState);
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <FormTitle>Add new post</FormTitle>
+      <div>
+        <TitleInput
+          type="text"
+          name="title"
+          placeholder="Enter title..."
+          value={postState.title || ''}
+          onChange={handleChange}
+        />
+        <Icon>🌨</Icon>
+      </div>
+      <MessageInput
+        type="text"
+        name="message"
+        placeholder="Enter message..."
+        value={postState.message || ''}
+        onChange={handleChange}
+      />
```

Comment:
Do inputów dodał bym jeszcze 'required' czy jakoś tak, żeby nie dało się zrobić submita zanim się nie wpisze tytułu i treści.

Reply:
Tak, tak, mam to w TODO jeszcze.

Reply:
Done